.circleci/config.yml

```diff
@@ -316,15 +319,18 @@ jobs:
     environment:
       SERVICE_NAME: text-summarization
     steps:
-      - checkout
-      - build_and_test_service:
-          service_name: text-summarization
-          org: singnet
-          docker_org: singularitynet
-          run_test: false
-      - persist_to_workspace:
-          root: ~/singnet
-          paths: text-summarization
+      - when:
+          condition: true
```

Comment:
I'm not quite sure what `condition: true` does here?

Reply:
I've added this in case user just wants to build/deploy one of the SIX(!) services.
So the user doesn't have to wait all the other 5 builds/deploys.
The elegant way will be using ENV_VARS from CircleCI here, but it's not supported yet:
[my-idea](https://ideas.circleci.com/ideas/CCI-I-980)

I have an idea on how to make it works with ENV_VARS but it's not elegant at all. =/
Something like [example-1](https://circleci.com/docs/2.0/configuration-reference/#example-1)

So for testing purpose (like build-deploys in user personal CI) I'd like to keep it as it is.
User will set all conditions that must not run to "false" and when he decides to open a PR he will have to set all back to "true".